apps/web-giddh/src/app/ledger/components/new-ledger-entry-panel/new-ledger-entry-panel.component.ts

```
@@ -1859,19 +1862,19 @@ export class NewLedgerEntryPanelComponent implements OnInit, OnDestroy, OnChange
-                        if (variantDiscount?.discount?.uniqueName === item?.uniqueName) {
-                            this.currentTxn.discounts.push(item);
-                        }
-                        return item;
-                    });
+        if (!this.currentTxn?.duplicateEntry) {
+            this.currentTxn.discounts = [];
+        }
+        const stockDiscounts = this.currentTxn.selectedAccount?.stock?.variant?.variantDiscount?.discounts
+        if (stockDiscounts?.length && !this.currentTxn.isMrpDiscountApplied) {
+            stockDiscounts?.forEach(variantDiscount => {
+                this.discountsList?.forEach(item => {
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)